src/transformers/__init__.py

```diff
@@ -5177,6 +5194,14 @@
             ViTMSNModel,
             ViTMSNPreTrainedModel,
         )
+
+        # PyTorch model imports
```

Comment:
Please don't move the "PyTorch model imports" line, should be at the top of the first model (albert)